src/docs/community/faq.mdx

```diff
@@ -1,36 +1,115 @@
+import { Divider, FAQExpander } from '../../app/components';
+
 <!-- KEYWORDS: faq question answers -->
 
 # Frequently Asked Questions
 
-### 1. How do I update to PX Blue 2.2+ if I'm currently using 2.1?
+We strive to provide as much information as possible about PX Blue in our documentation as possible. However, we can't always cover everything. Below are some common questions that people have asked about the platform. Please check to see if your question has been answered below before contacting us directly.
+
+## Developers
+
+<FAQExpander question={`How do I start a project?`}>
+
+We have comprehensive Getting Started guides for [Web](/development/frameworks-web/intro) and [Mobile](/development/frameworks-mobile/intro) projects that walk you through the steps necessary to add PX Blue to your project, including adding themes, components libraries, etc. If you are starting a new project, we highly recommend using the [PX Blue CLI](/development/cli) to start your project. This will automatically integrate all of the necessary PX Blue resources as well as give you several templates to choose from for starting your project.
```

Comment:
I would list CLI first and say "if you prefer to integrate manually, we have guides for x and x"